dist/stat_data.py

```diff
@@ -100,6 +100,12 @@ class SessionStat(Stat):
     prefix = 'session'
     def __init__(self, name, desc, flags=''):
         Stat.__init__(self, name, SessionStat.prefix, desc, flags)
+
+class PageStat(Stat):
```

Comment:
I don't think it should be called page stat at this level. We can discuss better naming.